fetcher.sh

```diff
@@ -146,9 +146,14 @@ Linux)
 		-e " terminator$" \
 		-e " termonad$" \
 		-e " x*st$" \
-		-e " tilda$")
+		-e " tilda$" \
+        -e " wezterm-gui$")
 	# remove leading space
 	term=${term# }
+    # if term doesn't appear, then check TERM_PROGRAM as some cross platform terms set it
+    if [ -z  "$term"]; then
+        term="$TERM_PROGRAM"
+    fi
```

Comment:
use

```
# if unset, fallback to $TERM_PROGRAM
term=${term:-$TERM_PROGRAM}
```

instead

Reply:
*might* (emphasis on the not-certain aspect) wanna duplicate that line for $TERM too, but a lot of terminals use it as a spec, ie st says `xterm-256color` because it's trying to support the same featureset as xterm-256color.

Reply:
ah coolio, ill do that as well